conn2res/task.py

```diff
@@ -127,47 +189,66 @@ def multiClassClassification(x, y, **kwargs):
     return score
 
 
-def multiOutputClassification(x, y, **kwargs):
+def multiOutputClassification(x, y, model=None, **kwargs):
     """
     Multiple output (binary and multi-class) classification tasks
-    #TODO
+    # TODO
     """
 
+    # pop variables from kwargs
+    sample_weight_train, sample_weight_test = kwargs.pop(
+        'sample_weight', (None, None))
+
     x_train, x_test = x
     y_train, y_test = y
 
-    model = MultiOutputClassifier(RidgeClassifier(alpha=0.5, fit_intercept=True, **kwargs)).fit(x_train, y_train)
-    score = model.score(x_test, y_test)    
-    
+    # specify default model
+    if model is None:
+        model = MultiOutputClassifier(RidgeClassifier(
+            alpha=0.5, fit_intercept=True, **kwargs))
```

Comment:
Again, as we discussed in our meeting, MultiOutputClassifier is actually not needed when used with RidgeClassifier as RidgeClassifier has the capability to perform multi-output classification off the shelf and gives the same result.

Another drawback of using MultiOutputClassifier is that is does not support sample weights. In contrast, had we used RidgeClassifier, it supports sample weights.